test/simulation/xud_test.go

```diff
@@ -137,7 +137,7 @@ func TestInstability(t *testing.T) {
 
 	for _, testCase := range instabilityTestCases {
 		success := t.Run(testCase.name, func(t1 *testing.T) {
-			ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
```

Comment:
How did you settle on 5 minutes here?

Reply:
I added it as a temporary step - 
The reason is that whenever a custom xud is being downloaded and installed, the test context lifetime is being exhausted, so the first RPC call will get a timeout. Best solution would be to extend the context whenever we're downloading/installing a custom xud, but I didn't want to deal with it in this PR.

Added to the roadmap.

Reply:
We can also define the required branches for the test suit before the test execution starts, but this might be a bit clumsy. I'll try it out when i'll work on it.